DEFI/ERC4626/runERC4626Full.conf

```diff
@@ -0,0 +1,12 @@
+{
+    "files": [
+        "certora/harnesses/mixins/ERC4626AccountingHarness.sol",
+        "certora/helpers/DummyERC20A.sol",
+        "certora/helpers/DummyERC20B.sol"
+    ],
+    "verify": "ERC4626AccountingHarness:certora/specs/ERC4626.spec",
+    "optimistic_loop": true,
+    "parametric_contracts" : ["ERC4626AccountingHarness"],
```

Comment:
need also the erc20

Reply:
If I understood you correctly, you wanted to add the helpers DummyERC20 as parametric contracts as well. If so that doesn't seem to pass all rules: https://vaas-stg.certora.com/output/15800/c3aea57ea9f54dcfbf02cb16a0d2a2ab?anonymousKey=a357df70c8c412c6338cc0cee8c38dfe5b708e6b

Reply:
1. prove that zero-allowance of current contract in the erc20. 
2. prove no dynamic calls
3. assume that e.msg.sender !=currentcontract.

Reply:
1. I proved like this: 
```rule zeroAllowanceOnAssets() {
    method f;
    env e;
    calldataarg args;

    require e.msg.sender != currentContract;

    address user;
    require ERC20a.allowance(currentContract, user) == 0;

    f(e, args);

    assert ERC20a.allowance(currentContract, user) == 0;
}```
Please let me know if that's ok.
2. What do you mean by dynamic calls?